tests/test_ble_lifecycle_controller.py

```
@@ -0,0 +1,689 @@
+
+        assert calls == ["scheduled"]
+
+
+class TestBLELifecycleControllerConnectionOwnershipDelegation:
+    """Test cases for connection ownership delegation methods."""
+
+    def test_emit_verified_connection_side_effects_delegates(self) -> None:
+        """_emit_verified_connection_side_effects should delegate to _connection_ownership."""
+        calls: list[object] = []
+        mock_ownership = SimpleNamespace(
+            _emit_verified_connection_side_effects=lambda client: calls.append(client)
```

Ruff (0.15.6)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)